WinImageBuilder.psm1

```diff
@@ -220,6 +228,7 @@ function Create-BCDBootConfig {
         [object]$image
     )
 
+    Write-Log "Create BCDBoot Config: $iamge"

```

Comment:
The rest of the comments do not respect the doing:finished paradigm like the one before.
Here it should be:
Write-Log "Creating BCDBoot config for $image.ImageName..."

Reply:
and after all has been created successfully, to have:
Write-Log "BCDBoot config has been created."